backend/codesirius/problems/models/execution_constraint.py

```
@@ -0,0 +1,80 @@
+                         This is measured in seconds.
+        memory_limit (int): The memory limit for the execution of the problem.
+                            This is measured in megabytes.
+        language (Language): The language for which the execution constraints are
+                            applied.
+    """
+
+    problem = models.ForeignKey(
+        Problem, on_delete=models.CASCADE, related_name="execution_constraints"
+    )
+    time_limit = models.IntegerField(
+        validators=[MinValueValidator(1), MaxValueValidator(300)], null=True
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)